src/main/java/ch/sbb/polarion/extension/pdf_exporter/service/PdfExporterPolarionService.java

```diff
@@ -115,7 +122,7 @@ private boolean isStylePackageSuitable(@Nullable String projectId, @NotNull Stri
             return true;
         } else {
             IDataService dataService = getTrackerService().getDataService();
-            IPObjectList<IModule> suitableDocuments =  dataService.searchInstances(dataService.getPrototype("Module"), model.getMatchingQuery(), "name");
+            IPObjectList<IModule> suitableDocuments = dataService.searchInstances(dataService.getPrototype("Module"), model.getMatchingQuery(), "name");
```

Comment:
replace `dataService.getPrototype("Module"")` to `IModule.PROTO`

Reply:
done